docs/adr/0011-operators.md

```diff
@@ -0,0 +1,153 @@
+# Managing operators
+
+- Status: accepted
+- Deciders: tumido, HumairAK, 4n4nd, anishasthana
+- Date: 2021-01-27
+
+Technical Story: [issue-1](https://github.com/operate-first/apps/issues/126), [pr-1](https://github.com/operate-first/apps/pull/131), [pr-2](https://github.com/operate-first/apps/pull/153)
+
+## Context and Problem Statement
+
+A `cluster-scope` application definition introduced by [ADR 0010][] is a bit vague for use cases like deploying operators. Operator Lifecycle Manager (OLM) defines a resource overlap where a single resource kind can be used in different contexts. For example, Subscription resource can subscribe to a specific operator which can be either cluster-scoped or namespace-scoped. This may go against the distinction introduced in [ADR 0010][]. This document should provide guidance to how each of the resource kinds for OLM should be treated in each context and usage variant we aim to support.
```

Comment:
Replace "this document *should* provide guidance" with "this document provides guidance" (because it it doesn't do that, we shouldn't merge it).

It should probably be "guidance on" rather than "guidance to".